source/_integrations/tplink.markdown

```diff
@@ -40,7 +43,7 @@ There is currently support for the following device types within Home Assistant:
 
 See [Supported Devices in python-kasa](https://github.com/python-kasa/python-kasa#supported-devices) for an up to date list.
 
```

Comment:
Move the note about unsupported versions here.

Reply:
Done